doc/workshop/conda.rst

```diff
@@ -15,8 +15,8 @@ What is Conda
 --------------------------------------------------------------------------------
 
 Conda is an open source package management system and environment management system that runs
-on Windows, macOS and Linux. Conda quickly installs, runs and updates packages and their dependencies.
-Conda easily creates, saves, loads and switches between environments on your local computer.
+on Windows, macOS, and Linux. Conda quickly installs, runs, and updates packages and their dependencies.
```

Comment:
I think this isn't correct, actually, but it doesn't much matter and is absurdly pedantic:

https://www.grammarly.com/blog/comma-before-and/

Reply:
I'll remove them and submit another PR.